src/handler/get_wallet_address.c

```diff
@@ -34,88 +35,109 @@
 
 #include "lib/policy.h"
 #include "lib/get_preimage.h"
+#include "lib/get_merkle_leaf_element.h"
 
 #include "get_wallet_address.h"
 #include "client_commands.h"
 
-static void compute_address(dispatcher_context_t *dc);
+bool parse_and_validate_wallet_policy(dispatcher_context_t *dc);
```

Comment:
I think this is never used

Reply:
Removed in fab57724f7d0aa6a1b40fa2bf5566c72275ad15c; while I was there, I also deleted all the `handler_xxx.h` files that became redundant after the refactoring.